packages/components/src/components/SquiggleViewer/utils.ts

```diff
@@ -43,13 +58,27 @@ export function extractSubvalueByPath(
   if (!value.context) {
     return;
   }
+  const { context } = value;
 
   for (const key of path.items) {
     let nextValue: SqValue | undefined;
-    if (typeof key === "number" && value.tag === "Array") {
-      nextValue = value.value.getValues()[key];
-    } else if (typeof key === "string" && value.tag === "Record") {
-      nextValue = value.value.get(key);
+    if (key.type === "number" && value.tag === "Array") {
+      nextValue = value.value.getValues()[key.value];
+    } else if (key.type === "string" && value.tag === "Record") {
+      nextValue = value.value.get(key.value);
+    } else if (key.type === "cellAddress" && value.tag === "TableChart") {
+      // Maybe it would be better to get the environment in a different way.
+      const environment = context.project.getEnvironment();
+      const item = value.value.item(
```

Comment:
Ah, I expected you'll implement `VTable.get()` method in squiggle-lang, but you did it this way instead.

I guess it's fine. Having `table.item(row, column)` in squiggle-lang might be useful to add later.

Reply:
I don't feel like I understand exactly what you mean here. All the variables in value/index.ts, like VPlot and VDomain, would only have get() methods that are Values. Here, we also need an environment, so this feels a bit different to what I see there. 

I imagine you might be able to fix this pretty quickly, if you ever want to. I'll leave for now.

Reply:
Huh, I didn't consider an environment, that's an interesting problem.

I think I could solve it by going through `SqValue`, which usually has access to the environment because `SqValueContext` has a link to the project, but it is a complication.